transforms/csdl/preprocess_csdl.xsl

```diff
@@ -708,6 +720,18 @@
         </xsl:copy>
     </xsl:template>
 
+    <!-- Remove indexability for joinedGroups navigation property -->
+    <!-- If the annotation already exists modify it -->
```

Comment:
Can we apply this logic to all existing nav properties to avoid future breaking changes?

Reply:
Thanks. Will resolve in next update.

Reply:
Resolved in latest update.